nginx.upstream.conf

```diff
@@ -1,23 +1,14 @@
-upstream summonerexpert {
-  # Each service gets an entry in /etc/hosts with its name.
-  # the app service is used to run the app and starts the server on port 3000
-  server app:3000;
-}
-
 server {
   listen 80 default_server deferred;
   listen [::]:80 default_server deferred;
 
-  root /app/public;
   access_log /etc/nginx/log/access.log;
   error_log /etc/nginx/log/error.log info;
 
-  try_files $uri @summonerexpert;
-
-  location @summonerexpert {
-    proxy_set_header Host $host;
-    proxy_set_header X-Forwarded-Proto $scheme;
+  root /app;
+  try_files $uri /index.html;
 
-    proxy_pass http://summonerexpert;
+  location ~* ^/(service-worker.js|index.html)$ {
+    add_header Cache-Control "max-age=0, no-cache, no-store, must-revalidate";
```

Comment:
Can we add a comment with a link to where it talks about needing this for service-workers?